src/main/java/com/powsybl/openloadflow/lf/outerloop/AbstractAreaInterchangeControlOuterLoop.java

```diff
@@ -267,4 +246,13 @@ private Map<String, Double> allocateSlackDistributionParticipationFactors(LfNetw
         return areaSlackDistributionParticipationFactor;
     }
 
+    @Override
+    public OpenLoadFlowParameters.SlackDistributionFailureBehavior getSlackDistributionFailureBehavior(O context) {
+        OpenLoadFlowParameters.SlackDistributionFailureBehavior slackDistributionFailureBehavior = context.getLoadFlowContext().getParameters().getSlackDistributionFailureBehavior();
+        if (OpenLoadFlowParameters.SlackDistributionFailureBehavior.DISTRIBUTE_ON_REFERENCE_GENERATOR == slackDistributionFailureBehavior) {
+            logger.error("Distribute on reference generator is not supported in AcAreaInterchangeControlOuterLoop, falling back to FAIL mode");
```

Comment:
The log mentions AcAreaInterchangeControlOuterLoop, but we are in the abstract class so we could also be in a DC AIC loop

Reply:
Fixed in [2c979b1](https://github.com/powsybl/powsybl-open-loadflow/pull/1123/commits/2c979b1a0230fb2d7485702af79f924a36f879f8) (wrong commit message)